Program.cs

```diff
@@ -1,10 +1,16 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿// Fibonacci series
+// Given an input eg:n=10, print fibonacci series for f(10)
+
+int first_number = 0;

```

Comment:
Variable name should not have underscore. You can do the initialization of same type variable in a single line.